assets/src/edit-story/components/inspector/document/publish/publishTime.js

```diff
@@ -119,6 +119,7 @@ function PublishTime() {
           aria-pressed={showDatePicker}
           aria-haspopup={true}
           aria-expanded={showDatePicker}
+          aria-label={__('Edit: Story publish time', 'web-stories')}
```

Comment:
@barklund There are some cases where even though the label exists, it doesn't seem to be necessarily accurate. E.g. here just "Publish" doesn't seem to be informative enough. In these cases, it seems that it still makes sense to have a specified label instead of using `aria-labelledby`.

The same goes for some other cases, e.g. Overlay buttons where it seems to be useful to specify that it sets the specific overlay (e.g. as opposed to toggling).

Read this about the labelling:
> In the cases where a visible label or visible tooltip is undesirable, authors MAY set the accessible name of the element using aria-label

Thoughts?

Reply:
Yes, in such a case it's perfectly okay to add manual labels. For the sighted user, the layout of the input and the context gives additional information than is given by the label alone - which is not always available to a non-sighted user.